src/AppInitializer.tsx

```diff
@@ -31,217 +35,245 @@ export type SignInProps = {
 
 const COOKIE_CONFIG_KEY = 'org.sagebionetworks.security.cookies.portal.config'
 
-class AppInitializer extends React.Component<Props, AppInitializerState> {
```

Comment:
I have no idea why I was getting "Invalid use of hooks" before converting this to a functional component and using hooks instead of passing props with HOCs, but the error went away once I changed it.